helper/m4/configure.m4

```diff
@@ -8,14 +8,13 @@ AC_ARG_ENABLE([test-helper],
 AM_CONDITIONAL([test_helper], [test x$test_helper = xyes ])
 
 ##########################################################################
-# Enable/disable helper-ext
-# platform specific non portable extensions
+# Enable/disable Linux helpers
 ##########################################################################
-helper_linux=no
+helper_linux=yes
 AC_ARG_ENABLE([helper-linux],
-	[  --enable-helper-linux	build helper platform extensions (not portable)],
-	[if test "x$enableval" = "xyes"; then
-		helper_linux=yes
+	[  --enable-helper-linux   enable Linux helpers],
```

Comment:
if you enable it by default ./configure --help should print --disable-helper-linux   disable Linux helpers.

Reply:
I think all config options we have added should be documented as enable-xxx. The help output is harder to read when some options are documented as enable-xxx (do xxx) and others disable-xxx (don't do xxx). The disable option is available in any case.

I trust only on the defaults printed by ./configure, not the help documentation which may be outdated.

Reply:
I think no. You have some option by default. They might be on or off. ./configure can change default settings. And help should print right information about that. --enable for already enabled option makes no sense.

Reply:
Compare e.g. autoconf enable shared/static library configs:

--enable-shared[=PKGS]  build shared libraries [default=yes]
--enable-static[=PKGS]  build static libraries [default=yes]

I did the same in v3
--enable-helper-linux   enable Linux helpers [default=yes]

I think we could do the same for all our configs in help text: --enable-xxx [default=yes/no]
--disable-xxx is always there for those that want to use that instead of --enable-xxx=no